components/EventsCard.tsx

```diff
@@ -0,0 +1,52 @@
+import AspectRatio from '@mui/joy/AspectRatio';
+import Card from '@mui/joy/Card';
+import CardContent from '@mui/joy/CardContent';
+import CardOverflow from '@mui/joy/CardOverflow';
+import Divider from '@mui/joy/Divider';
+import Typography from '@mui/joy/Typography';
+import Image from 'next/image';
+import Button from '@mui/joy/Button';
+import FacebookIcon from '@mui/icons-material/Facebook';
+import { Box } from '@mui/joy';
+
+// Hardedcoded for now
+// TODO (rbeccahsu): Work on integrating website with Contentful
+
+export default function EventsCard() {
+  return (
+    <Card 
+      variant="outlined" 
+      sx={{
+        width: 600,
+        '&:hover': { boxShadow: 'md', borderColor: 'neutral.outlinedHoverBorder' },
```

Comment:
Theres a .shadow class in global.css that Sophia made, might be better to use it for consistency?